internal/provider/data_source_gitlab_project_issues_test.go

```diff
@@ -14,7 +14,7 @@ func TestAccDataSourceGitlabProjectIssues_basic(t *testing.T) {
 	testProject := testAccCreateProject(t)
 	testIssues := testAccCreateProjectIssues(t, testProject.ID, 25)
```

Comment:
Note for other reviewers (if any choose to check this out 😄  )- even though CreatProjectIssues doesn't clean up after itself, the issues will be deleted when the project does.